app-starter/README.md

```diff
@@ -0,0 +1,8 @@
+# Wegue App Dir
```

Comment:
Shouldn't this be placed in `app` instead of `app-starter`? Or is this here because this file is copied by `init:app` script?

Reply:
Yes, good suggestion. The intention was that it would be the README for the app dir. That may be somewhat confusing placing it in `app-starter`. But as nothing is overwritten during copy, I can move it to `app` and have a separate README in `app-starter`. Hold on...